File: Entities/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BookStore.Entities
{
    public class Book
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string SubTitle { get; set; }
        public string CoverPicture { get; set; }
        public string Description { get; set; }
        public string Author { get; set; }
        public float Price { get; set; }
        public float Discount { get; set; }
        public Category Category { get; set; }
    }


}

File: ClientApp/src/app/_models/book.ts
import { Category } from "./category";

export class Book {
  id: number;
  title: string;
  subTitle: string;
  coverPicture: string;
  description: string;
  author: string;
  price: number;
  discount: number;
  category: Category;
  totalQuantity: number;
  orderQuantity: number;
}
